index.html

```diff
@@ -0,0 +1,134 @@
+            <div><a href="#" class="mylogo">My logo</a></div>
+            <nav class="hamburger"><img src="Assets/files/hamburger.png" alt="loading"></nav>
+        </header>
+        <section class="home-body">
+            <h1 class="heading">I'm Denyse. Glad to see you!</h1>
+            <p class="paragraph">I'm a software developer!I can help you build a product, feature or website. Look through some of my work and experience! If you like waht you see and have a project you need coded, dont hesitate to contact me.</p>
+            <div class="connect">LET'S CONNECT</div>
+            <ul id="social-links">
+                <li><a href="#"></a><img src="Assets/files/assets/twitter.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/Linkedin icon.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/medium.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/github.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/Vector.jpg" alt=""></li>
+            </ul>
+        </section>
+    </section>
+    <section>
+        <ul  id="works">
+            <li class="container1">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot.png" alt=""></li>
+                    <li class="works-heading">Tonic</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <button>See project</button>
+                </ul>
+            </li>
+            <li class="container2">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot2.png" alt=""></li>
+                    <li class="works-heading">Multi-Post Stories</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <button>See project</button>
+                </ul>
+            </li>
+            <li class="container1">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot3.png" alt=""></li>
+                    <li class="works-heading">Tonic</li>
```

Comment:
- [ ] As the previous reviewer pointed out, the project title (e.g., "Tonic") uses a header tag **hx**. This will enhance the semantics of your website. Kindly remedy that detail 🙏🏽